sdk/security/keyvault/azkeys/CHANGELOG.md

```diff
@@ -1,11 +1,9 @@
 # Release History
 
-## 1.0.0 (unreleased)
+## 0.12.0 (unreleased)
```

Comment:
I thought this was finally GA'ing? Or are you doing one final RC?

Reply:
Talked it over with Rick today and want to do one final beta since we're changing so much of the surface area.